src/components/Toolbar/LinkToolbar.js

```diff
@@ -183,9 +183,12 @@ class LinkToolbar extends React.Component<Props, State> {
     href = href.trim();
 
     if (href) {
+      //If the input starts with 'mailto:', keep it the same.
       // If the input doesn't start with protocol or relative slash, make sure
       // a protocol is added
-      if (!href.startsWith("/") && !href.match(/^https?:\/\//i)) {
+      if (href.startsWith("mailto:")) {
+        href;
+      } else if (!href.startsWith("/") && !href.match(/^https?:\/\//i)) {
```

Comment:
I actually think the entire change in this PR could be?

```suggestion
      } else if (!href.startsWith("/") && !href.match(/^(https|mailto)?:\/\//i)) {
```

Reply:
What would this 'else if' satement do? Are you just trying to ignore mailto with it? If so, it doesn't work as a single if statement. I tried. It only ignores https. 
Did you mean to also remove this code (to keep it as a single if statement)?:

`      if (href.startsWith("mailto:")) {
        href;
      }`

 or would you keep my added code? 

On a side note, break and continue were creating errors in the if function so I just recalled href instead. I know its not clean but I am open to suggestions.

Reply:
Ah yes, it wouldn't work as I suggested because `mailto` links aren't in the format `mailto://` – your latest solution seems fine 👍